app/src/main/java/com/example/habitsmasher/HabitEvent.java

```diff
@@ -11,11 +12,17 @@
  * Pictures for habit events are not yet implemented
  */
 public class HabitEvent extends DatabaseEntity implements Serializable {
+
     private Date _date;
     private String _comment;
     private Uri _pictureUri;
 
-    // TODO: Eventually add location
+    /*
+    String in the format of "latitude longitude" that stores the location
+    of the habit event, is an empty string if no location specified
+     */
+    private String _location;
```

Comment:
This field stores the location of the habit event. Since the location is optional, if the location is not specified, it is still stored as an empty string